azure-pipelines.yml

```diff
@@ -630,30 +705,49 @@ jobs:
   # Do not run this job for Pull Requests due to the slowness
   condition: and(succeeded(), in(variables['Build.Reason'], 'IndividualCI', 'BatchedCI', 'Manual', 'Schedule'))
   pool:
-    name: ${{ variables.ubuntu_pool }}
+    name: $(pool)
   strategy:
     matrix:
-      Red Hat Universal Base Image 8:
-        dockerfile: ubi
-        image: registry.access.redhat.com/ubi8/ubi:8.4
-        artifact: rpm-ubi8
-        python_package: python39
-      Red Hat Universal Base Image 9:
-        dockerfile: ubi
-        image: registry.access.redhat.com/ubi9/ubi:9.0.0
-        artifact: rpm-ubi9
-        python_package: python3.9
-      Fedora35:
-        dockerfile: fedora
-        image: fedora:35
-        artifact: rpm-fedora35
-        python_package: python3
-      Fedora36:
-        dockerfile: fedora
-        image: fedora:36
-        artifact: rpm-fedora36
-        python_package: python3
+      ${{ each arch in parameters.architectures }}:
```

Comment:
Azure pipeline's expression is so powerful, and `each` keyword can be used in matrix.
https://learn.microsoft.com/en-us/azure/devops/pipelines/process/expressions?view=azure-devops#each-keyword
https://github.com/microsoft/azure-pipelines-yaml/issues/20#issuecomment-491349961